setup.py

```diff
@@ -1,32 +1,52 @@
-import os
-import sys
+from setuptools import setup
 
 try:
```

Comment:
Pattern we're now using in other projects - have a README.md which is converted to RST for the long_description using pypandoc